pkg/render/manager.go

```diff
@@ -1166,3 +993,183 @@ func managerClusterWideDefaultView() *v3.UISettings {
 		},
 	}
 }
+
+// ManagerBasePolicyRules defines the RBAC rules required for manager pods, as well as for guardian identity to handle
+// manager requests originating from the management cluster.
+func ManagerBasePolicyRules(isManagedCluster, isOpenShift bool, tenant *operatorv1.Tenant) []rbacv1.PolicyRule {
+	rules := []rbacv1.PolicyRule{
+		{
+			APIGroups: []string{"authorization.k8s.io"},
+			Resources: []string{"subjectaccessreviews"},
+			Verbs:     []string{"create"},
+		},
+		{
+			APIGroups: []string{"authentication.k8s.io"},
+			Resources: []string{"tokenreviews"},
+			Verbs:     []string{"create"},
+		},
```

Comment:
Are you sure these are needed in managed clusters?

Reply:
I’m not certain about that. If we’re confident these aren’t required in managed clusters, I can add a !managedCluster check accordingly